### PR TITLE
Fixed significant mistake in FAQ about contract deployment

### PR DESCRIPTION
**Pull request by ProgramCrafter**
1. The contracts do not use any private keys for deployment.
2. Mentioned ton-solidity project.
3. Mentioned that wallets support sending several messages at once, allowing a mini-batch of outgoing messages.
4. Added Q+A about contract deletion.
5. Mentioned dton.io among RPC providers.

### CODE REVIEW

File: docs/develop/howto/faq.md [hunk 1]
```diff
@@ -111,6 +111,8 @@ Yes, transaction batching on TON can be accomplished in two distinct ways:
 Example of using batch-featured contract (high-load wallet):
 - https://github.com/tonuniverse/highload-wallet-api
 
+Default wallets (v3r2/v4r2) also support sending multiple messages in one transaction.
```
Comment (Gusarich) [inline docs/develop/howto/faq.md:114]: Let's remove the "r2" part from version numbers. Just 'v3/v4".

File: docs/develop/howto/faq.md [hunk 2]
```diff
@@ -214,9 +218,11 @@ Yes, smart contract addresses are case sensitive because they are generated usin
 
 ### Is it possible to write on Solidity for TON?
 
-  Relatedly, the TON ecosystem does not support development in Ethereum’s Solidity programming language.
+  Recently, the TON ecosystem did not support development in Ethereum’s Solidity programming language.
+  
+  The tool to write TON contracts in Solidity is in development. See it on Github: https://github.com/tonlabs/TON-Solidity-Compiler
```
Comment (Gusarich) [inline docs/develop/howto/faq.md:223]: I'm not sure if we should put this in documentation. And even if we should, at least add some warning that this tool is being developed by some 3rd party and we recommend using the languages and tools mentioned in docs.

**Review by Gusarich — CHANGES_REQUESTED** [review]
overall LGTM, but needs a few changes (check the review comments)

File: docs/develop/howto/faq.md (hunk 2 above)
Comment (ProgramCrafter) [reply docs/develop/howto/faq.md]: I think there is some chance that eventually this tool will become official, so I think it should be in documentation. Added warning that it is experimental and may work incorrectly.

**Review by Gusarich — APPROVED** [review]
LGTM

**Pull request merged**